homeassistant/components/zwave_js/light.py

```diff
@@ -283,7 +290,7 @@ async def async_turn_on(self, **kwargs: Any) -> None:
             await self._async_set_colors(rgbw_channels, transition)
 
         # set brightness
-        await self._async_set_brightness(kwargs.get(ATTR_BRIGHTNESS), transition)
+        await self._async_set_brightness(kwargs.get(ATTR_BRIGHTNESS, self._last_known_brightness), transition)
```

Comment:
this is what I meant in my feedback in the other discussion. Do not set the value of last known brightness as you're overriding the last known brightness of the device here